src/modules/shared/types/index.ts

```diff
@@ -10,3 +10,12 @@ export interface ComponentLink {
 	to: string;
 	external?: boolean;
 }
+
+export type SortOrder = 'asc' | 'desc';
+
+export enum Breakpoints {
+	sm = 576,
+	md = 768,
+	lg = 992,
+	xl = 1200,
+}
```

Comment:
Gebruiken in https://github.com/viaacode/hetarchief-client/blob/b56c3eb6552c709c13527978295dc40fe06e720e/src/modules/shared/components/MediaCardList/MediaCardList.const.ts

Reply:
Done